### PR TITLE
Update completion screens

### PR DESCRIPTION
**Pull request by Davidslv**
Story: https://mojdigital.teamwork.com/#/tasks/20230248

## What Next

Screenshot 2020-04-29 at 09 18 42
Screenshot 2020-04-29 at 09 18 34
Screenshot 2020-04-29 at 09 18 27

### Before (panel)

Screenshot 2020-04-29 at 08 49 19

### After

Screenshot 2020-04-29 at 09 21 08
Screenshot 2020-04-29 at 09 21 27

### CODE REVIEW

File: app/views/steps/completion/confirmation/show.en.html.erb [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-    <div class="grid_content_header govuk-box-highlight">
-      <h1 class="app__heading heading-xlarge gv-u-heading-xxlarge">Your application has been submitted</h1>
-      <p class="display-block util_mt-none util_mb-none">Your reference code is:</p>
-      <h3 class="heading-medium util_mt-none util_mb-medium"><%= current_c100_application.reference_code %></h3>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-panel govuk-panel--confirmation">
+      <h1 class="govuk-panel__title">
+        Your application
+        <br>
```
Comment (zheileman) [inline app/views/steps/completion/confirmation/show.en.html.erb:8]: Let's to get rid of this `<br>` as they don't work nice with different screen sizes and I also believe they are not recommended for screen readers.

File: app/views/steps/completion/confirmation/show.en.html.erb [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
+        has been submitted
+      </h1>
+      <div class="govuk-panel__body">
+        Your reference code is:
+        <br>
```
Comment (zheileman) [inline app/views/steps/completion/confirmation/show.en.html.erb:13]: However here, so the text does not wrap too much, we can remove the `<br>` but use:
<span class="app-util--no-wrap">Your reference code is:</span>

File: app/views/steps/completion/confirmation/show.en.html.erb [hunk 3]
```diff
@@ -34,7 +40,7 @@
       </p>
 
       <% unless user_signed_in? || current_c100_application.receipt_email? %>
-        <div class="panel panel-border-narrow">
+        <div class="govuk-inset-text">
           <p>If you want to keep a copy, you must download it now. You cannot return to a submitted application.</p>
```
Comment (zheileman) [inline app/views/steps/completion/confirmation/show.en.html.erb:44]: No paragraphs inside inset-text
https://design-system.service.gov.uk/components/inset-text/

File: app/views/steps/shared/_under_age_section.html.erb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<p>
+<p class="govuk-body">
```
Comment (zheileman) [inline app/views/steps/shared/_under_age_section.html.erb:1]: The list in this partial need the new style classes. Also, as usual the `govuk-link` class in the links.

File: app/views/steps/completion/confirmation/show.en.html.erb [hunk 5]
```diff
@@ -17,12 +25,10 @@
...
-      <%= render partial: 'steps/shared/payment_instructions', locals: {
-        c100_application: @c100_application
-      } %>
-    </div>
+    <h2 class="govuk-heading-l">Pay the application fee</h2>
+    <%= render partial: 'steps/shared/payment_instructions', locals: {
+      c100_application: @c100_application
+    } %>
 
     <%= render partial: 'under_age_section' %>
```
Comment (zheileman) [inline app/views/steps/completion/confirmation/show.en.html.erb:34]: There are parts not yet migrated, like the following div, h2 and paragraph.

File: app/views/steps/completion/confirmation/show.en.html.erb [hunk 6]
```diff
@@ -34,7 +40,7 @@
       </p>
 
       <% unless user_signed_in? || current_c100_application.receipt_email? %>
-        <div class="panel panel-border-narrow">
+        <div class="govuk-inset-text">
           <p>If you want to keep a copy, you must download it now. You cannot return to a submitted application.</p>
         </div>
       <% end %>
```
Comment (zheileman) [inline app/views/steps/completion/confirmation/show.en.html.erb:46]: The button to download the PDF is not migrated, use link_button helper (look for an example that also uses GA tracking as it needs the classes and data-module)

File: app/views/steps/completion/what_next/show.en.html.erb [hunk 7]
```diff
@@ -1,81 +1,74 @@
...
-              <a href="https://get.adobe.com/uk/reader/" rel="external" target="_blank">Adobe Acrobat Reader</a>.
-            </p>
-
-            <% unless user_signed_in? %>
-              <div class="panel panel-border-narrow">
-                <p>To avoid losing your application, you must download it now. You cannot return to a completed application.</p>
-              </div>
-            <% end %>
+    <h2 class="visuallyhidden">Steps</h2>
+    <div class="govuk-govspeak gv-s-prose">
```
Comment (zheileman) [inline app/views/steps/completion/what_next/show.en.html.erb:16]: This wrapper must go

File: app/views/steps/completion/what_next/show.en.html.erb [hunk 8]
```diff
@@ -1,81 +1,74 @@
...
-              This will be a PDF. If you cannot open the PDF after downloading, try using
-              <a href="https://get.adobe.com/uk/reader/" rel="external" target="_blank">Adobe Acrobat Reader</a>.
-            </p>
-
-            <% unless user_signed_in? %>
-              <div class="panel panel-border-narrow">
-                <p>To avoid losing your application, you must download it now. You cannot return to a completed application.</p>
-              </div>
-            <% end %>
+    <h2 class="visuallyhidden">Steps</h2>
```
Comment (zheileman) [inline app/views/steps/completion/what_next/show.en.html.erb:15]: It is now `govuk-visually-hidden`

File: app/views/steps/completion/what_next/show.en.html.erb [hunk 9]
```diff
@@ -1,81 +1,74 @@
...
+    <div class="govuk-govspeak gv-s-prose">
+      <ol class="list list-number app-list--icon-steps">
```
Comment (zheileman) [inline app/views/steps/completion/what_next/show.en.html.erb:17]: List is not fully migrated, it contains old classes.

File: app/views/steps/completion/what_next/show.en.html.erb [hunk 10]
```diff
@@ -1,81 +1,74 @@
...
+                        data: {ga_category: 'completion', ga_label: 'download application'} %>
+          </p>
+        </li>
 
-          <% if current_c100_application.applicants.under_age? %>
-            <li class="util_mt-medium">
-              <h3 class="heading-medium">
-                Send certificate of suitability or court order appointing a litigation friend
-              </h3>
+        <li class="util_mt-medium">
```
Comment (zheileman) [inline app/views/steps/completion/what_next/show.en.html.erb:39]: All the `util_mt-medium` must go, these are old classes to add margins to the top, with the new design system we should not need them.
For an example of numbered (with icons) list, take a look for example to:
`app/views/entrypoint/v1.en.html.erb`

**Comment by zheileman** [comment]
I think that should be all.

Please double check @Davidslv in case I've missed something.

Screen Shot 2020-04-29 at 12 51 55
--
Screen Shot 2020-04-29 at 12 51 14

File: app/views/steps/completion/what_next/show.en.html.erb [hunk 11]
```diff
@@ -1,81 +1,63 @@
...
-      <h2 class="visuallyhidden">Steps</h2>
-      <div class="govuk-govspeak gv-s-prose">
-        <ol class="list list-number">
-          <li class="util_mt-medium">
-            <h3 class="heading-medium">Download your application</h3>
-            <p>
-              This will be a PDF. If you cannot open the PDF after downloading, try using
-              <a href="https://get.adobe.com/uk/reader/" rel="external" target="_blank">Adobe Acrobat Reader</a>.
-            </p>
+    <h2 class="govuk-visually-hidden">Steps</h2>
```
Comment (Davidslv) [inline app/views/steps/completion/what_next/show.en.html.erb:15]: ah! I was looking for this stuff, thanks!

**Comment by Davidslv** [comment]
Ah! I can't approve my own PR, @zheileman when you ready 👍

**Pull request merged**